src/guides/bioinformatics/index.rst

```diff
@@ -21,3 +21,4 @@ How-to guides for performing bioinformatic anaylses with Nextstrain.
    fasta_input
    seq_traits
    examples
+   filtering-and-subsampling
```

Comment:
Open to suggestions for where to place this in the existing order. I don't think it should be at the very end, but unsure where would be best.

Reply:
I'd vote to place it before the [Augur and snakemake page](https://nextstrain--192.org.readthedocs.build/en/192/guides/bioinformatics/augur_snakemake.html). 

The last part of the filtering/subsampling page that alludes to connecting to `augur align`

> If you are building a standard Nextstrain workflow, the output files should be used as input to sequence alignment

flows nicely into the example on the "Augur and snakemake" page that connects `augur filter` and `augur align`.

Reply:
Done: c942ae6dcf95a464228eeb02153157e4d62e4615